config-laptop.plist/kaby-lake.md

```diff
@@ -83,7 +83,10 @@ Settings relating to ACPI, leave everything here as default as we have no use fo
 
 ![Booter](../images/config/config-universal/aptio-iv-booter.png)
 
-This section is dedicated to quirks relating to boot.efi patching with OpenRuntime, the replacement for AptioMemoryFix.efi
```

Comment:
Ahh I completely blanked on this!
This line should be put back.

Reply:
No worries, I'll do that right now.